src/app/content/highlights/utils/index.ts

```diff
@@ -14,3 +16,14 @@ export {
   updateSummaryHighlightsDependOnFilters,
   getSortedSummaryHighlights
 } from './summaryHighlightsUtils';
+
+export const getHighlightScrollTarget = (object: {[key: string]: any}, hash: string): HighlightScrollTarget | null => {
+  if (hash && object.type === 'highlight' && typeof object.id === 'string') {
```

Comment:
It is supposed to work with any object because this is what `state.query` in navigation is like
I could change it to accept only an object with specific properties, but then I would have to make another util to check if any object has these properties